go.mod

```diff
@@ -0,0 +1,11 @@
+module github.com/papertrail/go-tail
+
+go 1.16
```

Comment:
Any reason for picking `go 1.16` specifically? Otherwise I'd suggest doing this using the latest Go release.

(Or if you'd prefer me to redo it using the latest version, let me know and I'd be happy to do that!)

Reply:
It's compatible with older versions, and forward-compatible with never versions. Nevertheless, I use 1.19 anyway, I updated it to use 1.19.